data_structures/doubly_linked_list.py

```diff
@@ -47,16 +43,13 @@ def insert_ele(self, val):
         p.prev.succ = q
         q.succ = p
         p.prev = q
-    return self.head, self.tail
 
   def delete_ele(self, val):
-    if self.head is None:
-      raise DoublyLinkedListError("An empty list is here! No nodes could be deleted.")
```

Comment:
Consider this: if your list is non-empty, but does not contain the value to delete, then this function does not do anything. So to be consistent, we should also just do nothing if the list is empty.